src/Button.tsx

```diff
@@ -88,39 +215,80 @@ export const Button: React.FC<Props> = ({
     }
   }
 
+  /**
+   * Define the padding here so we can use it for both padding-right and
+   * padding-left
+   */
+  const paddingY = size === "small" ? 8 : size === "large" ? 8 : 7;
+
   return (
     <button
       {...otherProps}
       disabled={disabled}
+      onClick={event => {
+        if (onClick) {
+          onClick(event);
+        }
+
+        // We want to hide the blue border around a button after we've clicked
+        // on it.
+        event.currentTarget.blur();
+      }}
       css={[
         {
           "&[disabled]": {
-            backgroundColor: colors.silver.dark,
-            color: colors.grey.light,
+            backgroundColor:
+              feel === "flat"
+                ? "transparent"
+                : theme === "light"
+                ? colors.silver.dark
+                : colors.grey.dark,
+            color:
+              feel === "flat" && theme === "dark"
+                ? colors.grey.dark
+                : colors.grey.light,
 
             // We need to also set the `:hover` on `:disabled` so it has a higher
             // specificity than any `:hover` classes passed in. This also means
             // that both of these need to be overriden if we want to use a custom
             // disabled color.
             ":hover": {
-              backgroundColor: colors.silver.dark,
-              color: colors.grey.light,
+              backgroundColor:
+                feel === "flat"
+                  ? "transparent"
+                  : theme === "light"
+                  ? colors.silver.dark
+                  : colors.grey.dark,
+              color:
+                feel === "flat" && theme === "dark"
+                  ? colors.grey.dark
+                  : colors.grey.light,
             },
           },
         },
 
         {
           backgroundColor:
```

Comment:
This object and the one above it in the array could be merged together:

```js
[
  {'&[disabled]': ...},
  {backgroundColor: ...},
  // ...other stuff
]
// can be...
[
  {
    '&[disabled]': ...,
    backgroundColor: ....,
    // ...other keys
  },
  // ...other objects
]
```

Reply:
@cheapsteak explicitly separated them out, but I don't remember why. I think I'll leave it for now. This whole thing could use a clean-up, but it should be ready to ship now.

Ship when it's useful, not perfect!

Reply:
This is the file from when I separate them  
https://github.com/apollographql/space-kit/blob/f2ed4b77f16d6412a9ec0dc0ce5d01abdea45dd2/src/Button.tsx

At the time there was a conditional `variant === "simple" && { backgroundColor: "transparent" },` separating the 1st block and the 3rd block. 

https://github.com/apollographql/space-kit/blob/f2ed4b77f16d6412a9ec0dc0ce5d01abdea45dd2/src/Button.tsx#L105-L108

It looks like that's been removed so this should be good to merge into one object

Reply:
Thanks @cheapsteak 	! Always good to get context from the source. Since this hasn't been approved by anyone I'll make the change.

>non blocking of course

Very much appreciate when people write this.